frontend/src/pages/question-service/ViewQuestionPage.tsx

```diff
@@ -1,11 +1,70 @@
-import { useParams } from "react-router-dom"
+import { useParams } from "react-router-dom";
+import PageHeader from "@/components/common/PageHeader";
+import { fetchQuestion } from "@/api/question-service/QuestionService";
+import { useEffect, useState } from "react";
+import { Question } from "@/api/question-service/Question";
+import Difficulty from "@/components/question-service/Difficulty";
 
 export default function ViewQuestionPage() {
   const params = useParams();
+  const id = params.id as string;
+  const [question, setQuestion] = useState<Question | null>(null);
 
-  const id = params.id;
+  useEffect(() => {
+    const loadQuestion = async () => {
+      if (id) {
+        const ques = await fetchQuestion(id);
+        setQuestion(ques);
+      }
+    };
+    loadQuestion();
+  }, [id]);
 
   return (
-    <>View question page with ID {id}</>
-  )
-}
\ No newline at end of file
+    <>
+      <PageHeader />
+      {!question ? (
+        <div className="container mx-auto py-10 px-4">
+          I'm afraid there's no question here dear
```

Comment:
HASHAHAAHAH

Reply:
one more thing, in this case, it would be nice to add a link directing users to the questions page:

```
import { Link } from "react-router-dom";

<Link to="/questions">
  Go back to questions
</Link>
```